dss/collections/__init__.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from dss import dynamodb  # type: ignore
+
+
+collection_db_table = f"dss-collections-db-{os.environ['DSS_DEPLOYMENT_STAGE']}"
+
+
+def put_collection(owner: str, uuid: str, permission_level: str = 'owner'):
+    assert permission_level in ('owner', 'read-only')
+    dynamodb.put_item(table=collection_db_table,
+                      hash_key=owner,
+                      sort_key=uuid,
+                      value=permission_level,
+                      dont_overwrite='sort_key')
+
+
+def get_collection(owner: str, uuid: str) -> str:
```

Comment:
Just a note, this is unused, but will be used for sharing collections.  I can take it out if necessary and add it later when necessary.  @xbrianh Let me know.